packages/assets-controllers/src/selectors/token-selectors.ts

```diff
@@ -137,7 +152,13 @@ const selectAllEvmAccountNativeBalances = createAssetListSelector(
       for (const [accountAddress, accountBalance] of Object.entries(
         chainAccounts,
       )) {
-        const accountGroupId = accountsMap[accountAddress];
+        const account = accountsMap[accountAddress.toLowerCase()];
```

Comment:
`toLowerCase()` here because `accountsByChainId` in mobile uses checksumed addresses, unlike every other controller state.

This is only temporary though, because as soon as native addresses are returned in token balances, all this code can go.